doc/GettingStartedKubernetes.md

```diff
@@ -467,38 +552,44 @@ provides more detail about how the app server interacts with Vitess.
 
 ## Tear down and clean up
 
-The following command tears down the Container Engine cluster. It is
-necessary to stop the virtual machines running on the Cloud platform.
+Before stopping the Container Engine cluster, you should tear down the Vitess
+services. Kubernetes will then take care of cleaning up any entities it created
+for those services, like external load balancers.
+
+``` sh
+vitess/examples/kubernetes $ ./guestbook-down.sh
```

Comment:
If you delete the cluster without first removing the services, Kubernetes leaves behind the randomly-named forwarding-rules and target-pools that it created. Those are annoying to clean up now that they're randomly-named, so I figure it's easier to just tell Kubernetes to do it.